<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Copy_Money</class>
 <widget class="QMainWindow" name="Copy_Money">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>857</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>휴먼둥근헤드라인</family>
           </font>
          </property>
          <property name="text">
           <string>내역</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="searched_coin"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="font">
             <font>
              <family>휴먼둥근헤드라인</family>
             </font>
            </property>
            <property name="text">
             <string>공지사항</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextBrowser" name="notice_2"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>start</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <family>휴먼둥근헤드라인</family>
             </font>
            </property>
            <property name="text">
             <string>코인 List</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="coin_list_cbox"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>휴먼둥근헤드라인</family>
           </font>
          </property>
          <property name="text">
           <string>관찰중인 코인</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="coinlist">
          <column>
           <property name="text">
            <string>코인명</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>현재가</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>진행상황</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <family>휴먼둥근헤드라인</family>
             </font>
            </property>
            <property name="text">
             <string>보유중인 코인</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="wallet">
            <column>
             <property name="text">
              <string>코인명</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>현재가</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>수익률 (%)</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>790</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
